addons/macrochanger/macrochanger.lua

```diff
@@ -153,16 +154,16 @@ windower.register_event('addon command', function(...)
 	if args[1] == 'disableall' then
 		if args[2] == 'on' then
 			globaldisable = 1
-			windower.add_to_chat(17, 'All automated macro switching disabled.')
+			warning('All automated macro switching disabled.')
```

Comment:
I would also make this a notice, it's not dangerous behavior, nor can it easily be triggered accidentally.